File: Tier2/src/main/java/via/sep3/group11/tier2/shared/DTOs/LoginDTO.java
package via.sep3.group11.tier2.shared.DTOs;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * Domain Transfer Object used to log in users in the system.
 * @version 27/11-2022
 * @author Group 11
 *
 */
@AllArgsConstructor
@Data
public class LoginDTO {

    private String email;
    private String password;

    // Added lombok instead of manually declaring stuff.
}

File: Tier1/Shared/DTOs/LoginDTO.cs
namespace Shared.DTOs;

public class LoginDTO
{
    public string Email { get; }
    public string Password { get; }

   // public string ErrorMessage { get; }

   /*
    public LoginDTO(string email, string password, string errorMessage)
    {
        this.Email = email;
        this.Password = password;
        ErrorMessage = errorMessage;
    }
    */
    
    public LoginDTO(string email, string password)
    {
        this.Email = email;
        this.Password = password;
    }
}